src/common/reports/report_service.go

```diff
@@ -127,38 +148,43 @@ func (r *ReportService) CreateReport() *Report {
 // Updated Resources: <int>
 // <New Resources Table> as generated by printNewResourcesToStdout, if not empty
 // <Updated Resources Table> as generated by printUpdatedResourcesToStdout, if not empty
-func (r *ReportService) PrintToStdout() {
-	PrintBanner()
-	fmt.Println(colorReset, "Yor Findings Summary")
-	fmt.Println(colorReset, "Scanned Resources:\t", colorBlue, r.report.Summary.Scanned)
-	fmt.Println(colorReset, "New Resources Traced: \t", colorYellow, r.report.Summary.NewResources)
-	fmt.Println(colorReset, "Updated Resources:\t", colorGreen, r.report.Summary.UpdatedResources)
+func (r *ReportService) PrintToStdout(colorBool bool) {
```

Comment:
@goody-cvs Why replace the meaningful `noColor` name with a `colorBool` name?
Now when `colorBool` is `false`, it actually means there should be color 😓

Reply:
You have a very good point. I reverted that change. Hopefully the new names are more intuitive.